packages/core/src/common/command.ts

```diff
@@ -46,6 +46,13 @@ export interface Command {
      */
     category?: string;
     originalCategory?: string;
+    /**
+     * Default arguments for the command.
+     *
+     * Arguments specified in the command invocation are appended to this array.
+     */
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    arguments?: any[];
```

Comment:
@tsmaeder What are your thoughts on this change? I'm not sure where these changes are supposed to go otherwise.

Reply:
to me this seems to be just an additional feature for commands in general so why should this be a problem?

Reply:
FYI, I improved on this. Changes to the command/command registry are no longer necessary.

Reply:
ah ok nice. Looks good